pyntc/devices/eos_device.py

```diff
@@ -201,13 +233,23 @@ def uptime(self):
 
     @property
     def uptime_string(self):
+        """Create a string of the uptime of the device.
+
+        Returns:
```

Comment:
This property assigns only and does not explicitly return. I think it would be better to omit the `Returns` statement in the docstring.